src/core/stream/representation/get_needed_segments.ts

```diff
@@ -85,6 +89,13 @@ export interface IGetNeededSegmentsArguments {
  */
 const ROUNDING_ERROR = Math.min(1 / 60, MINIMUM_SEGMENT_SIZE);
 
+/**
+ * Minimum buffer (in seconds ) we should have, regardless of memory
+ * constraints
+ */
+const MIN_BUFFER_LENGTH = 5;
```

Comment:
Generally, we try to put those magic numbers in the `src/config.ts` file, even if it's true that it grew huge over time.

Reply:
Changed !